hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/BulkInsertFunctionWrapper.java

```diff
@@ -160,6 +160,13 @@ public void coordinatorFails() throws Exception {
     this.coordinator.setExecutor(new MockCoordinatorExecutor(coordinatorContext));
   }
 
+  public void coordinatorFailsAndRenewOne() throws Exception {
```

Comment:
maybe we rename it to `restartCoordinator` ?

Reply:
Addressed .PTAL.